core/src/main/java/io/grpc/internal/ServiceConfigUtil.java

```diff
@@ -59,6 +66,139 @@ static String getLoadBalancingPolicy(Map<String, Object> serviceConfig) {
     return null;
   }
 
+  static RetryPolicies getRetryPolicies(
```

Comment:
Maybe add Javadoc (or just a comment) that this will throw an exception if the service config doesn't parse?